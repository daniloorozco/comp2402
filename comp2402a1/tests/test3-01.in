Remove
and
return
the
object
at
position
index
in
the
list
Objects
at
positions
index
through
n-1
move
to
the
left
by
one
position
lao
the
hey
of
the
wer
by
remove
and
return
the
object
at
position
index
in
the
list
hey
at

index

n-1

to
the

by
one
position,

the
size
of

list
by
as




two
