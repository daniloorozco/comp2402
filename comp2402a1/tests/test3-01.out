return
object
position
the
positions
move
by
one
position
lao
